src/gles_fns.rs

```diff
@@ -1131,6 +1131,21 @@ impl Gl for GlesFns {
         }
     }
 
+    fn get_shader_precision_format(&self,
+                                   shader_type: GLuint,
+                                   precision_type: GLuint)
+                                   -> (GLint, GLint, GLint) {
+        unsafe {
+            let mut range = vec![0 as GLint; 2];
```

Comment:
nit: no need for heap allocation, no need to include these variable declarations as well as the result into `unsafe{}`